bin/install

```diff
@@ -1,115 +1,137 @@
-#!/bin/sh
+#!/usr/bin/env fish
 #
 # USAGE
-#   #1: curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | sh
-#   #2: curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install > install && chmod +x install && ./install
-#   #3: OMF_CONFIG=~/.omf curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | sh
+#   #1: curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | source -
+#   #2: curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install > install; and source ./install
+#   #3: env OMF_CONFIG=~/.omf curl -L github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | source -
```

Comment:
[here](https://github.com/oh-my-fish/oh-my-fish/pull/82/files#diff-81fd2adb13518cfcf88c004b912c4d9fR50) you have an `exit`. The exit will kill the current fish prompt so we might want to change `source` by `fish` :smile: